notebooks/notebooks/iclr2022/wr-and-margin-vs-training-steps.ipynb

```diff
@@ -431,76 +414,50 @@
t93PzuWwrYmkVQdGBdWknGSN3uXIg9VyB8LiiKgt1ux+12Z3WMxSQ9T/W8qc7TVG3M9Bqy3dftdjM2NhZ9PDQ0dNW8N4RYqkzvm9UkH68l1TGzvaYVUsFHvtfW1vKjH/0o6+1feumlrD+MhLhmlZSDurScliuRtsdqtWZ9S1uSngshxOpQ8J7KBx98kNbWVvr6+jKWbxsfH2fPnj0oikJDQ8MKtVAsZLFYuP322wFwOp3RnyPrgKTLViOLycDtN2yMe1z0FAVKltZLCbkNKmPPlcjjCLvdzujoaMZjLGU85cLnTXWeZmpjpvWZ9nU6nbhcrujjysrKq+a9IcRSZXrfrCb5eC2pjpntNa2Q8p78PJPx8XE2b96MTqfjwIEDfPrTn07YZmJigscee4xDhw4xNjaGoih0dHTwe7/3ewVo8bVLkp8XxuBUcHEphZaY7BzCvYIVFRVL2ncpRkZG8Hq9abeRpOdCCJEb+f4cL3hPpcPh4MiRI+zevZv9+/ezf/9+nE5nNF/lwMBA9HZ3JP5tbGyUgFKIVJaYRghW5tZ3LLvdnjGolHRCQgixOhR8TCVAQ0MDJ0+e5P7770fTNMbGxujr66Ovrw+32x1XSaexsZGnnnqqwC0WokgZLGBKP4wknZUOKo1GY9rn1Ol0kvRcCCFWiaK5WtfV1dHd3c0zzzzDnj174soyOp1OGhsb6ezs5JlnnsHhcBS6uUIUJ9vSSjJCuGrNYmdZ50K67A/SSymEEKtHwW9/L9TY2EhjY2OhmyHE6qPTg6VsybsXatyiXq/HarUyMzOTsE6CSiGEWD2KLqjM1tGjR7n//vsL3Yxr0pUrV3jkkUcA+NjHPsbXvva16LrIRKvPf/7zcctqampWtpE5cnl0gs9/8z+jjz/9m/dSU7a4vKorpqRqScnOI/IRVF6+fDmrc8Fut+PxeFg4b3CpQeXC5012nkbakamN6dZn2ve1117jiSeeiD5+6KGH4hIWr+b3hhBLle11YTXIx2tJdkwg62taIa3aoLKtrY0XX3yx0M24Jvn9fi5cuACA1+uN/gwQCAQAki5bjQKBEBeGxuMeFyVFBevSZ22rqpqXUoiBQCCrc0Gn01FSUsLU1FR02VKSnqd63lTnaTZtTLc+075erzdumcfjuWreG0IsVbbXhdUgH68l1TGzvaYV0qoMKh999FH6+voK3QxxtVKU8GQXsxO0pc+kXlEWF+iWHhSazeaCJxi32WzMzMwQCoUDd4PBUPA2CSGEyN6qCiq/8pWv0NbWJhV1Cqy0tJQPfehDAFRXV0d/BqIJ7JMtK1p6YziAtDjB7AKzI1qNxm4ej38t6+vAd6Uw7UxnGWmEgJzV+l7IbrdnfS6oqorNZmNiYgJY3njKhc+b6jzNpo3p1mfat7q6mu3bt0cfb9q0aXW9N4TIg8VcF4pdPl5LqmNme00rpIInP88kNvF5JL0QhG+NBYPBArfu2nJVJD+HcC+kxTkfSBqsizvO2CkYPJ7jxqWWMfm5yQ4V25Z8fEVRqKmpKYpeQU3TGBwcJBgMStJzIYTIsas++Xkqp0+fpr29nUOHDgHhDxtFUVAUJWEwvxBZ2XB7uBdSWWYmLVctqAa48ksohnNxGWmEoLhqayuKgt1ux+1252WMpxBCiPwpuqDy6NGjtLe309PTA8wHk5Gf6+rqcDqdvPTSS4VspliNLK7M22TLsR5UPVx6pbClH/WmcKC8DMXWG2i1WpmdnUWn0xW6KUIIIRahaJKff+UrX2Hr1q00NTXR09MTV0VH0zSam5vp7e3l5MmTHDt2jNra2gK3WFzz7DWwfnc4uCyUZfZSQvEFlQBOp7PQTRBCCLFIBe2pXDheEoi7ta0oCo888ggHDhxIqKLT0NCwkk0VIjlrOWy4Dc4fg6B/ZZ9b1YeffxkMBkNR9ghKaUYhhFh9ChJUJhsvGau5uZm6ujqeeOIJHn/88aTHuJbqf/f09NDR0UFfXx+jo6OUlZXR2NhIW1sbdXV1K96e8+fP85GPfASAT33qU3zhC1+Irjt48CAA+/fvj1u2fv36lW1kjpw/fz7zazE7YcMdcP5FCHhXrnHW8mWPD813L2VWv78VeN5k52mkHZnamG59pn2PHTsW97wLExav5veGEEtVqOtCPuTjtSQ7JpD1Na2QVrQ74OjRo7znPe+hvr6eQ4cOxd3idjqd7N+/n7GxMZ566iluu+22lWxa0WppaaGpqYm6ujr6+/sZGxuju7ubnp4e6uvr6erqKnQTBYRnYG+8E4yLnEm+VIqy7DRCUJy3voUQQqxOKxJUphsv2dDQQEdHB6Ojozz++OMJt7mvZS0tLXR1ddHa2kp7e3t0eV1dHb29vdFtIpOaRIEZrLDhznCAmW9mJ+iWVxdbp9PJDGshhBA5syJ5Kh944IFo4BOZyd3c3Myjjz7Kzp07k+7zrW99i717916zuSgPHTrEvn37ABgbG0s6caGtrY2DBw/idDoZGxvLe5si+a1qa2v5xje+AcCWLVs4efJkdJsdO3YA8Oqrr8Yts1pXqAcvx2ZmZhb/WoJ+uHAMPO6ctCFpnsrK68BoW9ZxS0pK8v4lbkm/vzw8b7LzNNKOTG1Mtz7TvsPDw3z/+9+PPr733nvjSqut5veGEEtVqOtCPuTjtSQ7JpD1NS2dfOepXLHk56dOneLxxx/n8OHDAOzbt4+HH36YW265Jen213pQWV9fz8DAAI2NjXR3dyfdZmBggPr6egDa29vjxlvkw6pPfr6SQkG42AfTw8s+VEJQaSyByu2pd8hSeXl53irpCCGEKD75/hxfsTGVtbW1dHR0EAqF+PKXv0x3dzcNDQ3cdttt/Ou//utKNWNV6OnpYWBgAICmpqaU20VydgJ0dHSsRNNEtlQdrNsVTjuUayXLH0upquqyyiAKIYQQCxUkb0draysnT57khz/8IU6nkwcffJDy8nI+//nPR+v+Xss6OzujP2dKnbR7924g3GvZ19eX13aJRVJUWHNrOF
```

Comment:
What's the significance of this number? Consider assigning it to a constant to make it self documenting.